pkg/clusteragent/orchestrator/processor.go

```diff
@@ -136,6 +146,7 @@ func processReplicaSetList(rsList []*v1.ReplicaSet, groupID int32, cfg *config.A
 	}
 
 	log.Debugf("Collected & enriched %d replica sets in %s", len(rsMsgs), time.Now().Sub(start))
+	log.Tracef("Send %d from total of %d replica sets", len(rsMsgs), len(rsList))
```

Comment:
We can always talk about the Log Level. It is spammy but then again useful information.